bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,8 +1,8 @@
 #GoAndKickBallMapGoal
 $AlignedToGoal
     YES --> $BallKickArea
-        LEFT --> @Stop, @KickBallDynamic
-        RIGHT --> @Stop, @KickBallDynamic
+        LEFT --> @Stop, @StandAndWait + duration:1 , @KickBallDynamic
+        RIGHT --> @Stop, @StandAndWait + duration:1 ,  @KickBallDynamic
```

Comment:
is one second a good time, i would assume that it might be shorter

Reply:
One second seems good imo.